src/app/components/job-list/job-list.component.ts

```diff
@@ -12,22 +13,33 @@ import { WeatherService } from 'src/app/services/weather.service';
   styleUrls: ['./job-list.component.scss']
 })
 export class JobListComponent implements OnInit, OnDestroy {
-  // TODO - Pick locations that will have alerts prior to meeting!
-  public jobs: JobData[];
+  // public jobs: JobData[];
+  public jobs: any;
+  public jobsResponse: JobsResponse;
   private ngUnsubscribe: Subject<void> = new Subject<void>();
 
   constructor(private jobService: JobService, private router: Router, private weatherService: WeatherService) { }
 
   ngOnInit(): void {
-    this.jobs = this.jobService.getJobs();
-    this.jobService.setSelectedJob(-1); // In case the user navigates here when a job was already selected, we want to "forget" the job that was selected
-
-    this.subscribeToJobServiceList();
+    // this.jobs = this.jobService.getJobs();
+    // this.jobService.setSelectedJob(-1); // In case the user navigates here when a job was already selected, we want to "forget" the job that was selected
+
+    // this.subscribeToJobServiceList();
+    
+    // No take 3 because we aren't calling here, and want to keep listening in case somebody else calls again. Right?
+    this.jobService.getLoading().pipe(takeUntil(this.ngUnsubscribe)).subscribe({
+      next: (loading: boolean) => {
+        if(!loading && this.jobService.hasSuccessfullyCompleted()) {
+          this.jobsResponse = this.jobService.getResults();
+          console.log(this.jobsResponse);
+        }
+      }
+    });
   }
 
   ngOnDestroy(): void {
     this.ngUnsubscribe.next();
-    this.ngUnsubscribe.complete
+    this.ngUnsubscribe.complete();
```

Comment:
@looksliketyler can you check on this? I added the (); because I'm pretty sure it's supposed to be there, but you did not have it in.

Reply:
complete is a method. resolved.